apps/src/craft/designer/game/LevelMVC/LevelEntity.js

```diff
@@ -82,7 +82,9 @@ export default class LevelEntity {
                     entity = new BaseEntity(this.controller, type, identifier, x, y, facing);
 
             }
-            console.log('Create Entity type : ' + type + ' ' + x + ',' + y);
+            if (this.controller.DEBUG) {
+                console.log('Create Entity type : ' + type + ' ' + x + ',' + y);
```

Comment:
Indentation just to match what is currently in the file, will re-format en masse later.